test/functional/test_framework/wallet.py

```
@@ -182,46 +196,93 @@
         if self._mode == MiniWalletMode.RAW_P2PK:
             self.sign_tx(tx)
         elif self._mode == MiniWalletMode.RAW_OP_TRUE:
-            tx.vin[0].scriptSig = CScript([OP_NOP] * 24)  # pad to identical size
+            for i in range(len(utxos_to_spend)):
+                tx.vin[i].scriptSig = CScript([OP_NOP] * 24)  # pad to identical size
         elif self._mode == MiniWalletMode.ADDRESS_OP_TRUE:
-            tx.vin[0].scriptSig = CScript([CScript([OP_TRUE])])
+            for i in range(len(utxos_to_spend)):
+                tx.vin[i].scriptSig = CScript([CScript([OP_TRUE])])
+        else:
+            assert False
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)